### DISCUSSION TITLE
Any upcoming plans for a release?

### DISCUSSION THREAD

**Discussion by MarkJNZ** [vcsjones/AzureSignTool#134] [General]
Any upcoming plans for a release?
I'm keen for a release which includes the support for skipping already signed files which was included in PR: https://github.com/vcsjones/AzureSignTool/pull/125I

**Comment by vcsjones** [comment]
Yeah, since the current release is based on .NET Core 2.1, and 2.1 is end-of-life soon, I plan on getting a beta out in the next few days.

**Comment by istvan-pal-shapr3d** [comment]
Is there any progress with getting the next release out? Can you please share the plan?

**Comment by vcsjones** [comment]
I had intended to get a release out late August.

The major hold up right now is renewing my Code Signing Certificate. A few days ago I actually got it, only to realize [NuGet Package Signing doesn't support ECDSA](https://twitter.com/vcsjones/status/1437523324838498308?s=20).

So I need to re-issue my Code Signing request as RSA. This (if you follow the twitter thread) is still pending as of last night. My hope is to bug them on a support ticket today and figure out why a re-key is taking so long.

**Reply by istvan-pal-shapr3d** [reply]
Thanks for the quick response! Good luck with support!

**Comment by vcsjones** [comment]
Version 3.0.0 was released today on NuGet. Thanks all for the patience.
